LoopKitUI/Views/Settings Editors/TherapySettingExtension.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TherapySettingExtension.swift
```

Comment:
[nit] Prefer `TherapySetting+HelpScreen.swift` if the extension's purpose is constrained

Reply:
I'm not sure what the future holds for this file: I'm imagining more things may go in here, such as more navigation things.  I'd prefer to leave as-is until later.